src/app/mypage/edit/page.tsx

```diff
@@ -0,0 +1,163 @@
+    setInterestJobs(selectedTags);
+  };
+
+  const handleSkillsSelectComplete = (selectedTags: string[]) => {
+    setSkills(selectedTags);
+  };
+
+  const handleSubmit = () => {
+    const profileData = {
+      username: name,
+      skills,
+      interestJobs,
+      introduction: '',
+    };
+    updateProfileMutation.mutate(profileData);
+  };
+
+  const renderTags = (tags: string[], type: 'interestJobs' | 'skills', onOpenModal: () => void) => (
+    <ul className="flex flex-wrap gap-2">
+      {tags.map((tag, index) => (
+        <li key={index}>
+          <TagInput
+            value={tag}
+            colorTheme="gray"
+            buttonType="delete"
+            onClick={() => handleTagDelete(type, index)}
+          />
+        </li>
+      ))}
+      <li>
+        <TagInput
+          value={type === 'interestJobs' ? '관심 직무' : '보유 스킬'}
+          onClick={onOpenModal}
+          colorTheme="gray"
+          buttonType="add"
+        />
+      </li>
+    </ul>
+  );
+
+  if (isLoading) {
+    return <PageSpinner />;
+  }
+
+  if (isError) {
+    console.error('유저 데이터 가져오기 실패:', error);
+    return <div>유저 데이터를 로드하는 중 오류가 발생했습니다.</div>;
+  }
+
+  return (
+    <main className="container mx-auto flex min-h-screen flex-col items-center p-12">
+      <div className="flex w-full justify-end" style={{ width: '90%' }}></div>
+      <div className="flex w-full max-w-[1040px] flex-col items-center gap-4">
+        <span className="self-start text-gray-900 body6">프로필 수정</span>
+        <div className="flex w-full max-w-[1040px] flex-col gap-8 rounded-[30px] bg-white p-8">
+          <div className="mx-auto flex w-full max-w-[500px] flex-col gap-8">
+            <div className="flex flex-col gap-2">
+              <span className="text-gray-600 mobile1">이름</span>
+              <Input placeholder="이름" value={name} onChange={handleNameChange} />
+            </div>
+            <div className="flex flex-col gap-2">
+              <span className="text-gray-600 mobile1">메일 주소</span>
+              <Input placeholder="PRism@gmail.com" value={email} disabled />
+            </div>
+            <div className="flex flex-col gap-2">
+              <span className="text-gray-600 mobile1">관심 직무</span>
+              {renderTags(interestJobs, 'interestJobs', handleOpenSelectInterestJobsModal)}
+            </div>
+            <div className="mb-[60px] flex flex-col gap-2">
+              <span className="text-gray-600 mobile1">보유 스킬</span>
+              {renderTags(skills, 'skills', handleOpenSkillsModal)}
+            </div>
+            <div className="flex justify-center gap-4">
+              <Button
+                variant="outline"
+                className="border-1 w-[72px] border border-gray-700 text-gray-700"
+                onClick={() => window.history.back()}>
+                취소
+              </Button>
+              <Button
+                variant="default"
+                className="w-[72px] bg-purple-500 hover:bg-purple-600"
```

Comment:
```suggestion
                className="w-[72px] "
```

variant='defulat' 도 삭재하고, 위에 배경, hover색도 삭제 가능합니다
Button의 defualt variant 색이 보라색이에용

Reply:
✅ [c1999dc](https://github.com/PRism-Works/PRism-FE/pull/73/commits/c1999dcf1c40bae3fc25ef7640ae5ef4b49c35a5)
ui/button hover 컬러 변경했고 불필요한 클래스 제거했습니다!